src/types.ts

```diff
@@ -12,11 +60,36 @@ export const testIndicator = {
 }
 export type TestIndicator = keyof typeof testIndicator;
 
+export type BillingData = {
+    /** Running number per recipient for each bill transmitted */
+    datenaustauschreferenzJeEmpfaengerIK: Record<string, number>
+    /** Indicate whether this bill is a test or if it is real data */
+    testIndicator: TestIndicator
+    /** Kind of bill, see documentation of RechnungsartSchluessel */
+    rechnungsart: RechnungsartSchluessel
+    /** X.509 Certificate including public key, ASN.1 Syntax in DER format */
+    senderCertificate: ArrayBuffer
+    /** corresponds to public key in senderCertificate, PKCS8 in DER format */
+    senderPrivateKey: ArrayBuffer
+
+    rechnungsnummerprefix: string
+    /** Date the bill was created. If not specified, the date is "now" */
+    rechnungsdatum?: Date
+    /** For which month this bill is. Bills are transmitted by month. */
+    abrechnungsmonat: Date
+    /** An ascending number indicating a correction of an earlier version of this same bill.
+     *  0 or undefined if this is not a correction. */
+    korrekturlieferung?: number
+    /** Mandatory if rechnungsart != "1" */
+    abrechnungsstelle?: Institution
```

Comment:
does `sender` have the same meaning as `abrechnungsstelle`? If yes, maybe put into one type? Or otherwise, add a clarifying comment?

Reply:
Not necessarily the same meaning. For `rechnungsart` 2 and 3 they happen to be the same, but for `rechnungsart == 1` they are not, because the Leistungserbringer is the sender. There is a [method for determining the sender](https://github.com/coop-care/paid/blob/8e2672bf1bd83d7edd8a88645e2a84b5a6cbc9a9/src/transmission/utils.ts#L15-L22), but I will also clarify the comment here.